src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine machine1 = new Excavator();
+        Machine machine2 = new Bulldozer();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
        Machine bulldozer = new Bulldozer();
```
краще використовувати більш інформативний неймінг